src/gui/KM_InterfaceDefaults.pas

```diff
@@ -56,6 +56,9 @@   TKMUserInterfaceCommon = class
   OPT_SLIDER_MIN = 0;
   OPT_SLIDER_MAX = 20;
   MAX_SAVENAME_LENGTH = 50;
+  //Limit names length to fit interface width
+  MAX_MAPNAME_LENGTH = 22;
+  MAX_TRACKNAME_LENGTH = 18;
```

Comment:
Let's move these 2 consts to a module where they are used. No need to make them "globally" visible.
After that PR can be merged )